### DISCUSSION TITLE
More the 1 Ecoflow Delta Pro

### DISCUSSION THREAD

**Discussion by csc-arnstadt** [vwt12eh8/hassio-ecoflow#52] [Q&A]
Hi Community!
Is it possible to integrate more then 1 Ecoflow Delta Pro? I have 2 and see only 1 device (Entity). The checkbox "check for new entitys" is on. 
Beste regards.
Rainer

**Comment by Ne0-Hack3r** [comment]
> Hi Community! Is it possible to integrate more then 1 Ecoflow Delta Pro? I have 2 and see only 1 device (Entity). The checkbox "check for new entitys" is on. Beste regards. Rainer

Yes. I have been monitoring 2 DP for a while now. I ended up naming them DP1 and DP2 to keep track of the entities for each unit...
